src/JavaFileContents.cs

```diff
@@ -319,11 +319,22 @@ public void ReturnAnonymousClass(string anonymousClassDeclaration, Action<JavaCl
             Indent(() =>
             {
                 JavaClass javaClass = new JavaClass(this);
-                anonymousClassBlock.Invoke(javaClass);
+                anonymousClassBlock(javaClass);
             });
             Line("};");
         }
 
+        public void AnonymousClass(string anonymousClassDeclaration, Action<JavaClass> anonymousClassBlock)
+        {
+            Line($"{anonymousClassDeclaration} {{");
+            Indent(() =>
+            {
+                JavaClass javaClass = new JavaClass(this);
+                anonymousClassBlock(javaClass);
+            });
+            Line("}");
```

Comment:
I think if you change this to `Text("}");` that will solve your right parenthesis on the next line problem that I mentioned in a different comment.

Reply:
I'll go with that.